pkg/liquidity-mining/contracts/gauges/ChildChainGaugeFactory.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@balancer-labs/v2-interfaces/contracts/liquidity-mining/IChildChainGauge.sol";
+import "@balancer-labs/v2-pool-utils/contracts/Version.sol";
+
+import "../VotingEscrowDelegationProxy.sol";
+import "./BaseGaugeFactory.sol";
+
+contract ChildChainGaugeFactory is Version, BaseGaugeFactory {
+    address public immutable votingEscrow;
```

Comment:
Make this private with a leading underscore, and add `getVotingEscrow`

Reply:
Removed, since now it's immutable and set in the gauge implementation directly.